mteb/tasks/Retrieval/eng/IFIRCdsRetrieval.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+
+class IFIRCds(AbsTaskRetrieval):
+    metadata = TaskMetadata(
+        name="IFIRCds",
+        dataset={
+            "path": "if-ir/cds",
+            "revision": "c406767",
+        },
+        description="BBenchmark IFIR cds subset within instruction following abilities.",
+        reference="https://arxiv.org/abs/2503.04644",
+        type="Retrieval",
+        category="s2p",
+        modalities=["text"],
+        eval_splits=["test"],
+        eval_langs=["eng-Latn"],
+        main_score="ndcg_at_20",
+        date=None,
```

Comment:
There are still a bunch of annotations missing. These will need to be filled out